### DISCUSSION TITLE
Downloading all student repositories

### DISCUSSION THREAD

**Discussion by abidmalikwaterloo** [community/community#63803] [GitHub Education]
### Hi everyone,

I am new to git education and planning to use it for my upcoming classes. I am testing/playing with some available resources. I installed gh on my fedora machine. However, getting the following message:

$ gh classroom clone student-repos -a 351557
unknown command "classroom" for "gh"

Why am I getting this? What is missing?

**Comment by denisbevilacqua** [comment]
If you haven't done so already, run `gh auth login` to authenticate with your GitHub account.

To install the GitHub Classroom extension, execute: 
```
gh extension install github/gh-classroom`.
```

To use gh to work with GitHub Classroom, type:
```
gh classroom SUBCOMMAND
```

**Comment by abhijit86k** [comment]
Should be

> ```
> gh extension install github/gh-classroom
> ```
